infrastructure/Dockerfile

```diff
@@ -12,10 +12,18 @@ RUN apt-get update && \
     apt-get install -y python3 python3-pip && \
     rm -rf /var/lib/apt/lists/*
 
+RUN apt-get update  \
```

Comment:
Why are we adding this?

Reply:
Updating the package is typically done  to ensure that the subsequent installation commands (apt-get install, for example) are working with the most up-to-date information about available packages.